src/components/Box/Box.tsx

```diff
@@ -10,11 +10,12 @@ import "./index.scss";
 export interface BoxProps {
   children: React.ReactNode;
   title: string;
+  dataQA?: string | null;
```

Comment:
I think we need change the props name to `dataQa` to keep it consistance with other components

Reply:
changed, thankyou